lib/bootloader_setup.pm

```diff
@@ -595,8 +590,8 @@ sub specific_bootmenu_params {
     }
 
     if (check_var("LINUXRC_KEXEC", "1")) {
-        $args .= " kexec=1";
-        record_soft_failure "boo#990374 - pass kexec to installer to use initrd from FTP";
+        $args .= " kexec=3";
```

Comment:
Does it make sense to use `LINUXRC_KEXEC` as binary flag after this change? Could we use it with value `3` and document it as well?

Reply:
I updated it with the suggestion. 

Also I added it in a way, that the kexec value is took from LINUXRC_KEXEC. So the value is propagated from openQA Test Suite settings. Next time if we'll need to send another kexec value, we could create new Test Suite without changing the source code.